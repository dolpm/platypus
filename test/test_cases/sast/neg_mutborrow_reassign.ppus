pipe main [] |> unit {

    int millie_bobbie_brown = 11;
    int david_harbour = 20;

    ~&int finn_wolfhard = ~&millie_bobbie_brown;
    finn_wolfhard = ~&david_harbour;

    |> ();
}