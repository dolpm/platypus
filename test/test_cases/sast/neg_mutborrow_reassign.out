Fatal error: exception Failure("mutborrow finn_wolfhard must be dereferenced to be assigned to")
